Documentation/ceph-object-store-crd.md

```diff
@@ -230,6 +233,8 @@ $ vault write -f transit/keys/<mybucketkey> exportable=true # transit engine
 
 * TLS authentication with custom certificates between Vault and CephObjectStore RGWs are supported from ceph v16.2.6 onwards
 
+* `serverSideEncryption` chooses type of [server side encryption](https://docs.aws.amazon.com/AmazonS3/latest/userguide/serv-side-encryption.html) supported for RGW, it can be set via boolean `kmsEnabled` for `kms` and `s3Enabled` for `s3`. If this option is not defined then RGW configured with `kms` options
```

Comment:
If encryption is used, kms is the default, correct? How about bullets with some clarification something like this? And do we need a comment about what it means to enable both of them?
```suggestion
* `serverSideEncryption` chooses type of [server side encryption](https://docs.aws.amazon.com/AmazonS3/latest/userguide/serv-side-encryption.html) supported for RGW:
  * `kmsEnabled`: Enable kms encryption (default is true)
  * `s3Enabled`: Enable s3 encryption (default is false)
```